spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -567,9 +570,9 @@ def save_as_csv(agg_metric, fname_out, fname_in=None, append=False):
         with open(fname_out, 'w', newline='') as csvfile:
             # spamwriter = csv.writer(csvfile, delimiter=',')
             header = ['Timestamp', 'SCT Version', 'Filename', 'Slice (I->S)', 'VertLevel', 'DistancePMJ']
-            agg_metric_key = [v for i, (k, v) in enumerate(agg_metric.items())][0]
+            first_metric_group = list(agg_metric.values())[0] if agg_metric else {}
```

Comment:
TIL the `next` builtin function allows specifying a default value:
```suggestion
            first_metric_group = next(iter(agg_metric.values()), {})
```